package.json

```diff
@@ -31,32 +33,43 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "yarn stripes test karma"
+    "test:unit": "jest --ci --coverage && yarn run test:unit:report",
+    "test:unit:report": "cp -r ./artifacts/coverage-jest ./artifacts/coverage",
+    "test:e2e": "yarn stripes test karma --karma.singleRun --karma.browsers ChromeDocker",
+    "test": "yarn run test:unit && yarn run test:e2e"
   },
   "devDependencies": {
+    "@babel/plugin-transform-runtime": "^7.0.0",
     "@bigtest/cli": "^0.2.2",
     "@bigtest/interactor": "^0.9.2",
     "@bigtest/mocha": "^0.5.2",
     "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^5.0.0",
-    "@folio/stripes": "^6.0.0",
     "@folio/stripes-cli": "^2.0.0",
+    "@folio/stripes": "^6.0.0",
+    "@testing-library/dom": "^7.29.6",
+    "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^11.0.2",
+    "@testing-library/user-event": "^12.7.1",
     "babel-eslint": "^10.0.0",
+    "babel-jest": "^26.3.0",
     "chai": "^4.2.0",
     "core-js": "^3.6.1",
-    "eslint": "^7.9.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.0.1",
+    "eslint": "^7.9.0",
```

Comment:
I doubt it's the right sorting. It looks like manual in vscode, but it's different from default sorting. try to add new modules through yarn commands, like yarn add modulename , after that you'll get generated package.json

Reply:
Right. Fixed it. Thx